src/TestSuite/IntegrationTestTrait.php

```diff
@@ -263,13 +270,27 @@ public function configApplication($class, $constructorArgs)
      * compatible token to be added to request data. This
      * lets you easily test actions protected by SecurityComponent.
      *
+     * @param string[] $unlockedFields List of fields that are excluded from field validation.
      * @return void
      */
-    public function enableSecurityToken()
+    public function enableSecurityToken($unlockedFields = [])
```

Comment:
why we need this argument when we can use `$this->setUnlockedFields($unlockedFields);` ??

Reply:
Agreed. Having both seems redundant.

Reply:
A good point. I can't think of a use case where `$this->setUnlockedFields($unlockedFields);` wouldn't do.